test-bitcoincore-rpc/src/lib.rs

```diff
@@ -395,6 +576,10 @@ impl Handle {
     let state = self.state();
     state.blocks[&state.hashes[bi]].txdata[ti].clone()
   }
+
+  pub fn mempool(&self) -> Vec<Transaction> {
+    self.state.lock().unwrap().get_mempool()
```

Comment:
Might also be `.into` or something else:

```suggestion
    self.state.lock().unwrap().mempool().to_vec()
```